rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala

```diff
@@ -112,7 +112,10 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
 
     // Check that there is some modification to enabled/disable save
     def check() = {
-      S.notice("updateChangeMsg","")
+      noModif match{
```

Comment:
On a boolean, you can do a if (noModif) {} else {} 

Ok i said it was better in scala to do pattern matching but not on everything ;)